src/Fable.Websockets.Suave/Fable.Websockets.Suave.fsproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\Meta.props" />
   <PropertyGroup>
     <TargetFramework>netstandard1.6</TargetFramework>
```

Comment:
This will need to change to v2 I believe

Reply:
we can keep TargetFramework minimal, until we need some new api